app/views/admin/merchants/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Admin Merchants</h1>
+<br>
+<%= link_to "Create New Merchant", "/admin/merchants/new" %>
+<br>
+<h3>Top 5 Merchants</h3>
+<ol>
+  <% Merchant.top_5.each do |merchant| %>
+    <section id="top_5-<%= merchant.id %>">
+      <li><%= link_to merchant.name,  admin_merchant_path(merchant) %>, Total Revenue Generated: <%= number_to_currency(merchant.revenue) %></li>
+    </section>
```

Comment:
Way to use the asset paths from the advanced routing, your code is looking clean